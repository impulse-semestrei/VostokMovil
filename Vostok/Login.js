import React from 'react';
import { StyleSheet, Text, View, ImageBackground, Dimensions, Button, Image,TouchableOpacity  } from 'react-native';


//FONDO DE LOGIN--
import background from './imagenes/home_background.jpg'

//logo
import logo from './imagenes/logo.png'
import boton from './imagenes/boton.jpeg'
const { width: WIDTH } = Dimensions.get('window');

export default class Login extends React.Component {
  render(){
    return (
      <ImageBackground source={background} style={styles.container}>

      <View style={styles.Imagecontainer}>
        <Image source={logo} style={styles.logo}>
        </Image>
      </View>

      <View style={styles.Botoncontainer}>
      <Image source={boton} style={styles.boton}>
      </Image>
      </View>

      </ImageBackground>
    );
  }
}


const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#f8f8ff',
    justifyContent: 'center',
  },
  backgroundContainer:{
    flex: 1,
    width: null,
    height: null,
    justifyContent: 'center',
    alignItems: 'center',
  },
  Imagecontainer: {
    alignItems: 'center',
    marginBottom: 50,
    marginLeft: 10,
  },
  logo: {
    width: 250,
    height: 80,
    resizeMode: 'stretch'
  },
  Botoncontainer: {
    alignItems: 'center',
    marginBottom: 100,
    marginHorizontal: 30,
  },
  boton: {
    width: 240,
    height: 50,
    resizeMode: 'stretch'
  },
  btnLogin :{
    width: WIDTH - 220,
    height: 45,
    borderRadius: 45,
    backgroundColor: 'rgba(220,20,60,50)',
    marginHorizontal: 109,
    marginTop: 30
  },
  text :{
  color: 'rgba(255,255,255,10)',
  fontSize: 16,
  textAlign: 'center',
  marginTop:10
},
});
