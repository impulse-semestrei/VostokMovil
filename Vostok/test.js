
import 'react-native';
import React from 'react';
import Login from './Login.js';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <Login />
    ).toJSON();
  expect(tree).toMatchSnapshot();
});
