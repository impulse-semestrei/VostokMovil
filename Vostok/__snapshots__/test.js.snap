// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "backgroundColor": "#f8f8ff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../imagenes/home_background.jpg",
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": undefined,
          "width": undefined,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "marginBottom": 50,
        "marginLeft": 10,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../imagenes/logo.png",
        }
      }
      style={
        Object {
          "height": 80,
          "resizeMode": "stretch",
          "width": 250,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "marginBottom": 100,
        "marginHorizontal": 30,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../imagenes/boton.jpeg",
        }
      }
      style={
        Object {
          "height": 50,
          "resizeMode": "stretch",
          "width": 240,
        }
      }
    />
  </View>
</View>
`;
